netty/src/main/java/io/grpc/netty/GrpcSslContexts.java

```diff
@@ -176,5 +187,15 @@ static void ensureAlpnAndH2Enabled(ApplicationProtocolNegotiator alpnNegotiator)
       throw new IllegalArgumentException("This ALPN config does not support HTTP/2. Expected '"
           + HTTP2_VERSION + "', but got " + alpnNegotiator.protocols() + '.');
     }
+    if (!alpnNegotiator.protocols().contains(GRPC_EXP_VERSION)) {
```

Comment:
I don't understand this check.  gRPC should still keep working even if GRPC_EXP_VERSION) is not present.